### PR TITLE
Stop syncing of a number of maven repositories

### PR DESCRIPTION
**Pull request by johnbintz-tidelift**

### CODE REVIEW

**Review by jhan217 — APPROVED** [review]

File: app/workers/package_manager_download_worker.rb [hunk 1]
```diff
@@ -39,12 +39,7 @@ def initialize(platform, name, version)
     inqlude: PackageManager::Inqlude,
     julia: PackageManager::Julia,
     maven: PackageManager::Maven,
-    maven_atlassian: PackageManager::Maven::Atlassian,
```
Comment (mikeyoung85) [inline app/workers/package_manager_download_worker.rb:42]: I _think_ this is only used when the worker is called with something like `maven_springlibs` and we aren't storing that value anywhere in the database. Does that sound right? I'm just trying to double check that removing these isn't going to break Project lookups for Maven projects that aren't in Central.

File: app/models/package_manager/maven/jboss_ea.rb [hunk 2]
```diff
@@ -1,18 +0,0 @@
-# frozen_string_literal: true
-
-class PackageManager::Maven::JbossEa < PackageManager::Maven::Common
```
Comment (mikeyoung85) [inline app/models/package_manager/maven/jboss_ea.rb:3]: I would probably hold off on removing these files until we can take a look through the data and verify we don't have any `Versions` pointing to these repository sources

File: app/workers/package_manager_download_worker.rb [hunk 3]
```diff
@@ -39,12 +39,7 @@ def initialize(platform, name, version)
...
-    maven_hortonworks: PackageManager::Maven::Hortonworks,
     maven_mavencentral: PackageManager::Maven::MavenCentral,
-    maven_springlibs: PackageManager::Maven::SpringLibs,
-    maven_jboss: PackageManager::Maven::Jboss,
```
Comment (tiegz) [inline app/workers/package_manager_download_worker.rb:46]: was there a reason for stopping JBoss? I still see about 3k updates from depper in the past month for `maven_jboss`.

File: docs/update-maven-repository-sources.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    * app/workers/package_manager_workload_worker.rb, `PLATFORMS`, add above class.
+* depper
+    * ingestors/maven.go
+      * add constant next to `MavenCentral`
+      * add to `GetParser()`
+    * main.go, add constant to `registerIngestors`
+* maven-updater
+  * src/m/j/i/l/m/services/, duplicate `MavenCentralUpdater.javaa and rework.
+    * The code uses reflection to find the new updater so you don't need to
+      change anything else.
```
Comment (tiegz) [inline docs/update-maven-repository-sources.md:17]: :thumbsup:  just FWIW, the process is similar for Conda and Packagist, IIRC

File: app/workers/package_manager_download_worker.rb (hunk 1 above)
Comment (tiegz) [reply app/workers/package_manager_download_worker.rb:42]: AFAIK it's only a mapping from the depper-provided platforms to these subclasses in Libraries

**Comment by johnbintz-tidelift** [comment]
I've talked this over with @mikeyoung85 and we're going to hold off on removing classes here until the syncing of the platforms has stopped on depper and the package download worker has stopped processing the removed sources.

**Review by tiegz — APPROVED** [review]

**Pull request merged**